drivers/bluetooth/hci/hci_ifx_cyw208xx.c

```diff
@@ -0,0 +1,357 @@
+/*
+ * Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+ * an affiliate of Cypress Semiconductor Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /**
+  * @brief CYW20829 driver.
+  */
+
+#include <errno.h>
+#include <stddef.h>
+#include <string.h>
+
+#include <zephyr/arch/cpu.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/drivers/bluetooth/hci_driver.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/init.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/logging/log.h>
+
+#include <wiced_bt_stack_platform.h>
+#include <cybt_platform_config.h>
+#include <cybt_platform_trace.h>
+#include <cybt_platform_hci.h>
+#include <cybt_platform_task.h>
+
+#include <cyabs_rtos.h>
+#include <cybt_result.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
+#define LOG_LEVEL  CONFIG_BT_HCI_DRIVER_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(cyw208xx);
+
+
+#define DT_DRV_COMPAT infineon_cyw208xx_hci
+
+
+#define PACKET_TYPE_HCI_COMMAND     0X1
```

Comment:
I think you can reuse values already defined (see https://github.com/zephyrproject-rtos/zephyr/pull/67401)

Reply:
removed